config/packages/test/lti1p3.yaml

```diff
@@ -4,5 +4,33 @@ lti1p3:
             key_set_name: "platformSet"
             public_key: "file://%kernel.project_dir%/config/secrets/test/public.key"
             private_key: "file://%kernel.project_dir%/config/secrets/test/private.key"
-            private_key_passphrase: null
-
+            private_key_passphrase: ~
+        toolKey:
+            key_set_name: "toolSet"
+            public_key: "file://%kernel.project_dir%/config/secrets/test/public.key"
+            private_key: "file://%kernel.project_dir%/config/secrets/test/private.key"
+            private_key_passphrase: ~
+    platforms:
+        testPlatform:
+            name: "Test platform"
+            audience: "https://localhost/platform"
+            oidc_authentication_url: "https://localhost/lti1p3/oidc/authentication"
+            oauth2_access_token_url: "http://localhost/lti1p3/auth/kid1/token"
```

Comment:
Same here about parametrizing host - it would be easier IMO to configure host only in .env file

Reply:
Migrated URLs to environment vars